cargo-sqlx/src/postgres.rs

```diff
@@ -89,7 +89,7 @@ impl DatabaseMigrator for Postgres {
 
         let mut conn = PgConnection::connect(base_url).await?;
 
-        sqlx::query(&format!("CREATE DATABASE {}", db_name))
+        sqlx::query(&format!("CREATE DATABASE \"{}\"", db_name))
```

Comment:
I was hesitant to just wrap the name in quotes because not all DB flavors treat identifiers the same way: https://stackoverflow.com/a/2901464

* MySQL quotes identifiers with backticks ``` ` ` ``` unless the `ANSI_QUOTES` option is enabled
* MS SQL (which we're working on) uses square brackets (?!) ` [ ] `

Reply:
We probably also want to forbid these characters appearing _in_ the database name, however unlikely.

Reply:
Of course for Postgres, the only thing we really care about is just forbidding double-quotes from appearing in the database name.